assets/js/src/form_editor/components/font_family_settings.tsx

```diff
@@ -155,14 +95,19 @@ const FontFamilySettings = ({
 
 export default FontFamilySettings;
 
-const customFontsUrl = customFonts
-  .map((fontName) => fontName.replace(' ', '+'))
-  .map((fontName) => fontName.concat(':400,400i,700,700'))
-  .join('|');
-
-export const CustomFontsStyleSheetLink = () => (
-  <link
-    rel="stylesheet"
-    href={`https://fonts.googleapis.com/css?family=${customFontsUrl}`}
-  />
-);
+export const CustomFontsStyleSheetLink = () => {
+  const customFonts = useSelect(
+    (select) => select('mailpoet-form-editor').getAllCustomFonts(),
+    []
+  );
+  const customFontsUrl = customFonts
+    .map((fontName) => fontName.replace(' ', '+'))
+    .map((fontName) => fontName.concat(':400,400i,700,700'))
```

Comment:
I see that on front end we have `:400,400i,700,700i` so maybe the `i` is missing here.